ibis/backends/bigquery/__init__.py

```diff
@@ -485,26 +485,31 @@ def from_connection(
         client: bq.Client,
         partition_column: str | None = "PARTITIONTIME",
         storage_client: bqstorage.BigQueryReadClient | None = None,
+        dataset_id: str = "",
     ) -> Backend:
         """Create a BigQuery `Backend` from an existing ``Client``.
 
         Parameters
         ----------
         client
-            A ``Client`` from the ``google.cloud.bigquery`` package.
+            A `Client` from the `google.cloud.bigquery` package.
         partition_column
-            Identifier to use instead of default ``_PARTITIONTIME`` partition
-            column. Defaults to ``'PARTITIONTIME'``.
+            Identifier to use instead of default `_PARTITIONTIME` partition
+            column. Defaults to `'PARTITIONTIME'`.
         storage_client
-            A ``BigQueryReadClient`` from the
-            ``google.cloud.bigquery_storage_v1`` package. If not set, one is
-            created using the ``project_id`` and ``credentials``.
+            A `BigQueryReadClient` from the `google.cloud.bigquery_storage_v1`
```

Comment:
Nit: I had these with double '`' to be consistent with the `do_connect` docstring; maybe I can update `do_connect` in a follow-up to be consistent (don't want to re-trigger CI now, just for a docs change).

Reply:
Yeah, that makes sense!